SharpDXtest/EditorWindow.xaml.cs

```diff
@@ -36,9 +37,14 @@ public EditorWindow()
 
         private void Window_Loaded(object sender, RoutedEventArgs e)
         {
+            var mainPath = Directory.GetCurrentDirectory();
+            var solutionPath = Directory.GetParent(mainPath)?.Parent?.Parent?.Parent?.Parent?.FullName;
+            AssetsManager.InitializeInFolder(solutionPath);
+            AssetsRegistry.InitializeInFolder(solutionPath);
+            
             EngineCore.Init(new WindowInteropHelper(this).Handle, (int)ActualWidth, (int)ActualHeight);
 
-            EngineCore.CurrentScene = AssetsManager_Old.LoadScene("Assets\\Scenes\\Scene5.xml");
+            CreateBaseScene();
```

Comment:
please recreate scene5 in a form of base asset and load it instead

Reply:
Another branch - tag components with SerializedField